### PR TITLE
x86_cpu_model: disable some unsupported models

### PR DESCRIPTION
**Pull request by nanliu-r**
ID: 2084388

Disable deprecated many named CPU models which may
have some problems while booting a RHEL9 guest.

Signed-off-by: nanliu <nanliu@redhat.com>

### CODE REVIEW

**Comment by nanliu-r** [comment]
@PaulYuuu Please help to review this, thanks a lot!

File: qemu/tests/cfg/x86_cpu_flags.cfg [hunk 1]
```diff
@@ -123,6 +123,7 @@
         - others:
             variants:
                 - test_default_model:
+                    no RHEL.9
```
Comment (PaulYuuu) [inline qemu/tests/cfg/x86_cpu_flags.cfg:126]: So what's the mean of the default model not supported in RHEL9? `qemu64`?

File: qemu/tests/cfg/x86_cpu_model.cfg [hunk 2]
```diff
@@ -202,13 +203,15 @@
                 flags = ""
         - Icelake-Client:
             only HostCpuVendor.intel
+            no RHEL.9
```
Comment (PaulYuuu) [inline qemu/tests/cfg/x86_cpu_model.cfg:206]: Is this also related to Host_RHEL?

File: qemu/tests/cfg/x86_cpu_flags.cfg (hunk 1 above)
Comment (nanliu-r) [reply qemu/tests/cfg/x86_cpu_flags.cfg:126]: > So what's the mean of the default model not supported in RHEL9? `qemu64`?

Yes, it is deprecated, and RHEL9 guest will crash with this model.

File: qemu/tests/cfg/x86_cpu_model.cfg (hunk 2 above)
Comment (nanliu-r) [reply qemu/tests/cfg/x86_cpu_model.cfg:206]: > Is this also related to Host_RHEL?

The original qemu bug is for virtualization part and we have no kernel bug to disble these platforms so the host should boot up successfully.

**Review by PaulYuuu — APPROVED** [review]

**Comment by PaulYuuu** [comment]
Seems no limited only to Linux, please consider adding a windows reviewer if needed.

**Comment by nanliu-r** [comment]
> Seems no limited only to Linux, please consider adding a windows reviewer if needed.

Thanks for reminding this, windows team has no CPU feature, and these old cpu models still work with windows guest.

 (1/2) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.x86_cpu_model.Opteron_G3.q35: STARTED
 (1/2) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.x86_cpu_model.Opteron_G3.q35: PASS (264.92 s)
 (2/2) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.x86_cpu_flags.others.test_default_model.q35: STARTED
 (2/2) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.x86_cpu_flags.others.test_default_model.q35: PASS (268.00 s)

There is no test for 'Icelake-Client' since there is a qemu version required 'required_qemu = (, 5.2.0)'.

Please help to review this, thanks.
